.github/workflows/memcheck.yml

```diff
@@ -248,4 +248,4 @@ jobs:
           name: heaptrack_safenode
           path: heaptrack.safenode.*
         continue-on-error: true
-        if: always()
\ No newline at end of file
+        if: always()
```

Comment:
prefer retain the last empty line.
i.e. the file diff shall not have that red stop sign.

Reply:
This file does not contain an empty last line, and I think this introduces that empty line. Not the other way around.